OnlineFlightSearchAPITestCases/RepositoryTests/UserRepositoryTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Moq;
+using OnlineFlightSearchAPI.DBModelsFolder;
+using OnlineFlightSearchAPI.Models;
+using OnlineFlightSearchAPI.Repositories;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace OnlineFlightSearchAPI.UnitTests.RepositoryTests
+{
+    public class UserRepositoryTests
+    {
+        [Theory]
+        [InlineData("abc", "123")]
+        public void FetchValidUserDetails_IfInValidUserNameAndPasswordProvided_ReturnsEmpty(string userName, string password)
+        {
+            var mockUsersDbSet = new Mock<DbSet<UserLoginModel>>();
+            var UsersList = new List<UserLoginModel> {
+                new UserLoginModel
+                {
+                    UserName=userName+"s",
+                    Password=password+"p"
```

Comment:
\+ ??

Reply:
Just to differentiate from Input User Model, appending those chars